src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookConnector.java

```diff
@@ -128,12 +153,33 @@ private boolean postKnowledgeTree(KnowledgeElement rootElement) {
 		return false;
 	}
 
+
+	private boolean postKnowledgeElement(KnowledgeElement changedElement) {
+		System.out.println("postKnowledgeElement");
+		WebhookContentProvider provider = new WebhookContentProvider(projectKey, changedElement, secret, receiver);
+		PostMethod postMethod = provider.createPostMethodForSlack(changedElement);
+		System.out.println(postMethod.toString());
+		try {
+			HttpClient httpClient = new HttpClient();
+			postMethod.setURI(new HttpsURL(url));
+			int httpResponse = httpClient.executeMethod(postMethod);
+			if (httpResponse >= 200 && httpResponse < 300) {
+				return true;
+			}
+			LOGGER.error("Could not send webhook data. The HTTP response code is: " + httpResponse);
+			//LOGGER.error("Could not send webhook data. The HTTP response code is: " + process);
+		} catch (IOException | IllegalArgumentException e) {
+			LOGGER.error("Could not send webhook data because of " + e.getMessage());
+		}
+		return false;
+	}
+
 	private boolean checkIfDataIsValid(KnowledgeElement changedElement) {
 		if (url == null || url.equals("")) {
 			LOGGER.error("Could not trigger webhook data because the url is missing.");
 			return false;
 		}
-		if (secret == null || secret.equals("")) {
+		if ((secret == null || secret.equals("")) && receiver != "Slack") {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)